functions/http/2.0/index.js

```diff
@@ -0,0 +1,46 @@
+import templayed from './templayed';
+
+const parseHeaders = (headers) =>
+  headers.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {});
+
+const parseQueryParameters = (queryParameters) =>
+  queryParameters
+    .map(({ key, value }, index) => {
+      const paramKey = index === 0 ? `?${key}` : key;
+      return `${paramKey}=${encodeURIComponent(value)}`;
+    })
+    .join('&');
+
+const generateUrl = (url, protocol, queryParameters) =>
+  `${protocol}://${url}${parseQueryParameters(queryParameters)}`;
+
+const https2 = async ({
```

Comment:
why call it https2 ?

Reply:
Whoopsie. The original step from Marcel is called https2! Forgot to adjust it here :(